themes-default/slim/views/partials/showheader.mako

```diff
@@ -223,16 +203,16 @@
                         <table class="pull-xs-left pull-md-right pull-sm-right pull-lg-right">
                             <% info_flag = subtitles.code_from_code(show.lang) if show.lang else '' %>
                             <tr><td class="showLegend">Info Language:</td><td><img src="images/subtitles/flags/${info_flag}.png" width="16" height="11" alt="${show.lang}" title="${show.lang}" onError="this.onerror=null;this.src='images/flags/unknown.png';"/></td></tr>
-                            % if app.USE_SUBTITLES:
-                            <tr><td class="showLegend">Subtitles: </td><td><img src="images/${("no16.png", "yes16.png")[bool(show.subtitles)]}" alt="${("N", "Y")[bool(show.subtitles)]}" width="16" height="16" /></td></tr>
-                            % endif
+                            <template v-if="config.subtitles.enabled">
+                                <tr><td class="showLegend">Subtitles: </td><td><img :src="'images/' + (show.config.subtitlesEnabled ? 'yes' : 'no') + '16.png'" :alt="show.config.subtitlesEnabled ? 'Y' : 'N'" width="16" height="16" /></td></tr>
+                            </template>
                             <tr><td class="showLegend">Season Folders: </td><td><img src="images/${("no16.png", "yes16.png")[bool(show.season_folders or app.NAMING_FORCE_FOLDERS)]}" alt="${("N", "Y")[bool(show.season_folders or app.NAMING_FORCE_FOLDERS)]}" width="16" height="16" /></td></tr>
```

Comment:
```vue
<tr><td class="showLegend">Season Folders: </td><td><img :src="'images/' + ((show.config.seasonFolders || config.namingForceFolders) ? 'yes' : 'no') + '16.png'" :alt="(show.config.seasonFolders || config.namingForceFolders) ? 'Y' : 'N'" width="16" height="16" /></td></tr>
```

Reply:
Couldn't remember where `namingForceFolders` was.

Reply:
I generally just do a search for `app.VAR_NAME` in the codebase and look for `medusa/server/api/v2/config.py` or open up that file and look for the variable.

Reply:
Btw an `||` doesn't need `()` around it in a ternary statement.

```js
((show.config.seasonFolders || config.namingForceFolders) ? 'yes' : 'no')
```

vs 

```js
(show.config.seasonFolders || config.namingForceFolders ? 'yes' : 'no')
```